client/syntaxes/coq.tmLanguage

```diff
@@ -36,7 +36,7 @@
 
         <dict>
             <key>match</key>
-            <string>\b(Require|Import|Export|Local|Global|Open|Scope)\b</string>
+            <string>\b(Require|Import|Export|Local|Global|Open|Scope|Context)\b</string>
```

Comment:
Why add `Context` here? TBH, I have no idea what this list was about in the first place.